platform/platform_freebsd.cpp

```diff
@@ -0,0 +1,142 @@
+#include "platform/platform.hpp"
```

Comment:
Exactly. We used virtual base class in 2010, then moved to current solution, but now I see that it was bad idea.

I think it would be great to refactor whole platform to use more clear virtual base class, rather than patch existing code (and people who port for other platforms will be grateful too). What do you think, is it feasible for you? (in a separate request, of course).

> On Oct 5, 2015, at 08:31, Dmitry Marakasov <notifications@github.com> wrote:
> 
> In platform/platform_freebsd.cpp <https://github.com/mapsme/omim/pull/112#discussion_r41158004>:
> 
> > @@ -0,0 +1,142 @@
> > +#include "platform/platform.hpp"
> , virtual base class would be much easier to handle. Now, as all stuff is initialized in ctor, you can't basically use virtual methods, so you have to parametrize base class ctor with e.g. executable path, which is non-flexible.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/112/files#r41158004>.
>